ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -129,6 +129,17 @@ ORK_CLASS_AVAILABLE
  */
 + (BOOL)removePasscodeFromKeychain;
 
+/**
+ This method allows to force the passcode to be a certain value. You only want to use this capability when you're moving your existing users
+ from a custom passcode entry mechanism to the one provided by ResearchKit.
+ 
+ This method will raise an exception if `passcode` is nil.
+ 
+ @param passcode          The passcode to store
+ @param touchIdEnabled    Whether TouchId will be available during passcode entry
+ */
++ (void)forcePasscode:(NSString *)passcode withTouchIdEnabled:(BOOL)touchIdEnabled;
```

Comment:
@p2 I would suggest move this API to the private interface(need to create) of `ORKPasscodeViewController`, at least we can hide it from main interface.